crates/typst/src/layout/page.rs

```diff
@@ -245,7 +248,7 @@ pub struct PageElem {
     /// The page's header. Fills the top margin of each page.
     ///
     /// - Content: Shows the content as the header.
-    /// - `{auto}`: Shows the page number if a `numbering` is set and
+    /// - `{auto}`: Shows the page number if the `numbering` is set and
```

Comment:
Either keep "a `numbering`" or explicitly write out "the `numbering` field". Not sure if the field can be linked to, if so it'd be preferable to keep the original with a link to the field.

Reply:
Now let's [go back](https://discord.com/channels/1054443721975922748/1088371867913572452/1292076710786760744) and settle the question I had for a long time. Is it field or parameter? I feel like in messages it's usually "field", but in the docs, IDs are named with "parameter". I think that we should streamline docs to 1 term to be more consistent and easier to identify what is being talked about. If they are interchangeable, then perhaps we can use both, but IDK.

But I also don't understand why you think that the original "a numbering" is equally good? The sentence mentions _the_ `page.numbering`, a specific thing, not some random `numbering` variable or something.

I think the link should be possible. I propose keeping the new change + adding the link to the `page.nubering`.

Reply:
- It's a field. "Parameter" is just a broad name for whatever you can use in a function or constructor, but for elements the constructor parameters are usually the same as the element's fields. Here there is no doubt that we're referring to the value of the field and not to the constructor.
- The original is good because it's referring to the word "numbering" itself. You can either have _a numbering_ be set on your page (i.e. have exactly one option of numbering format to be used in your page), or have _the numbering field_ (here the article refers to _field_ instead of _numbering_). Both work, which is why I don't think the change is necessary. But adding a link could be good.